src/views/FirstPage.vue

```diff
@@ -227,6 +230,7 @@
                     </div>
                   </div>
                 </b-card-body>
+                <WalletsTooltipsEthereum />
```

Comment:
I don't see the advantage of moving this out into a separate file. Popovers target specific elements in this template by ID, referencing those IDs from another file just makes it easier for cruft to build up when this component is modified.

Reply:
The login info age template looked like a monster. Too much clutter.